os-server/src/main/java/app/openschool/course/discussion/DiscussionForumController.java

```diff
@@ -38,17 +39,19 @@
 public class DiscussionForumController {
```

Comment:
isn't it possible to delete and update answers using only question and answer ids ?